tests/unit/test_data_validator.py

```
@@ -0,0 +1,81 @@
+    rec = OHLCVRecord(
+        ticker='A',
+        timestamp='2020-01-01',
+        open=10.0,
+        high=12.0,
+        low=9.0,
+        close=11.0,
+        volume=100,
+        vwap=0,
+    )
+    expected = OHLCVRecord._calculate_fallback_vwap(12.0, 9.0, 11.0, 100)
+    assert isinstance(rec.vwap, float)
```

Ruff (0.12.2)

Use of `assert` detected

(S101)